governance-exercises/Adjust Spork D3M Parameters and DAI Interest Rate -- July 24, 2023.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Adjust Spork Protocol D3M Parameters and DAI Interest Rate - July 24, 2023
+summary: Express your support or opposition to adjusting the Spork DAI Direct Deposit Module (D3M) parameters and DAI Interest Rate.
+discussion_link: https://forum.makerdao.com/t/xxx
+parameters:
+    input_format: single-choice
+    victory_conditions:
+        - { type : plurality }
+    result_display: single-vote-breakdown
+version: v2.0.0
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2023-07-24 16:00 UTC
```

Comment:
Format is strict for timestamps. These should be: 2023-07-24T16:00:00. All the times here are in UTC.